contracts/core/RebalancingSetToken.sol

```diff
@@ -199,9 +200,18 @@ contract RebalancingSetToken is
 
         // Make sure enough time has passed from last rebalance to start a new proposal
         require(block.timestamp >= lastRebalanceTimestamp.add(rebalanceInterval), "PROPOSE_CALLED_TOO_EARLY");
-
+        
         // Check that new proposed Set is valid Set created by Core
-        require(ICore(ISetFactory(factory).core()).validSets(_nextSet), "PROPOSED_SET_INVALID");
+        require(core.validSets(_nextSet), "PROPOSED_SET_INVALID");
+
+        // Check that the auction library is a valid priceLibrary tracked by Core
+        require(core.validPriceLibraries(_auctionLibrary), "PRICE_LIB_MUST_BE_VALID");
+
+        // Assert price divisor > 0
```

Comment:
These are really ineffective comments, provide some color here on why we do this